addons/l10n_uy/demo/account_demo.py

```diff
@@ -1,20 +1,613 @@
+            + self.ref('demo_invoice_4')
+            + self.ref('demo_invoice_5')
+            + self.ref('demo_invoice_6')
+            + self.ref('demo_invoice_8')
+            + self.ref('demo_invoice_9')
+            + self.ref('demo_sup_invoice_1')
+            + self.ref('demo_sup_invoice_2')
+            + self.ref('demo_sup_invoice_3')
+            + self.ref('demo_sup_invoice_6')
+            + self.ref('demo_sup_invoice_7')
+            + self.ref('demo_sup_invoice_8')
+            + self.ref('demo_sup_invoice_9')
+        )
+        # the invoice_extract acts like a placeholder for the OCR to be ran and
+        # doesn't contain any lines yet
+        for move in invoices:
+            try:
+                move.action_post()
+            except (UserError, ValidationError):
+                _logger.exception('Error while posting invoices')
+
+        # Post the reversal moves
+        invoices_to_revert = (
+            self.ref('demo_refund_invoice_1')
+            + self.ref('demo_refund_invoice_2')
+            + self.ref('demo_refund_invoice_3')
+            + self.ref('demo_refund_invoice_4')
+            + self.ref('demo_sup_refund_invoice_3')
+            + self.ref('demo_sup_refund_invoice_2')
+            )
+        for move in invoices_to_revert:
+            try:
+                self.env['account.move'].browse(
+                    move.refund_moves().get('res_id')).action_post()
+            except (UserError, ValidationError):
+                _logger.exception('Error while posting reversal moves')
+
     @api.model
     def _get_demo_data_move(self, company=False):
-        """ Set the l10n_latam_document_number on demo invoices """
-        move_data = super()._get_demo_data_move(company)
-        if company.account_fiscal_country_id.code == 'UY':
-            number = 1
-            for move in move_data.values():
-                if move['move_type'] in ('in_invoice', 'in_refund'):
-                    move['l10n_latam_document_number'] = 'A' + f'{number:07d}'
-                    number += 1
-
-        return move_data
+        cid = company.id or self.env.company.id
+        sale_journal = self.env['account.journal'].search(
+            domain=[
+                *self.env['account.journal']._check_company_domain(cid),
+                ('type', '=', 'sale')
+            ], limit=1)
+        purchase_journal = self.env['account.journal'].search(
+            domain=[
+                *self.env['account.journal']._check_company_domain(cid),
+                ('type', '=', 'purchase')
+            ], limit=1)
+        return {
+            # Customer invoice demo
+            'demo_invoice_1': {
+                'company_id': company.id,
+                'move_type': 'out_invoice',
+                'partner_id': 'base.res_partner_4',
+                'journal_id': sale_journal.id,
+                'invoice_payment_term_id':
+                    'account.account_payment_term_end_following_month',
```

Comment:
```suggestion
                'invoice_payment_term_id':  'account.account_payment_term_end_following_month',
```
Does not go over the line, so put it in a single line would be better, I think